_docs/project-setup.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "Project Setup: Extra Installations"
+sidebar-title: "Project Setup"
+---
+
+# Project Setup: Node And More
+{:.no_toc}
+
+So, you want to work on our cool projects like the Editor or a Learning Lab! We're excited to have you on our team, but there's a few things you have to setup before you can jump right into working on our projects!
+
+## Table of Contents
+{:.no_toc}
+* TOC
+{:toc}
+
+## Initial Setup
+
+Before we get started, make sure that you've already followed along with our [dev setup guide]({{ site.baseurl }}/docs/dev-setup) to get your terminal, git, and GitHub set-up!
+
+Our projects require a few more things we have to set-up as well, so be sure to head back to this page once you've set-up git and GitHub!
+
+## Installing node and npm
+
+For our projects, we use JavaScript and Typescript to handle website logic, so we'll also need additional tools and package managers to handle all our JavaScript dependencies. There are two primary package managers that we use, the first being the Node Package Manager, or npm. npm actually comes with `Node.js`, a JavaScript tool that we also need which is used to help build network applications, so let's check if we have it installed already! 
+
+Try running:
+
+```sh
+$ node -v
+```
+
+If you already have node installed in your machine, you should see some
+numbers in the form of `vXX.XX.X` which means that you already have node (and consequently npm) installed, and don't need to do anything else!
```

Comment:
Might want to add something about `npm -v` too!

Reply:
Also, we should probably mention something about using the right node version - esp with node 16, it really does matter which version they install.